index.html

```diff
@@ -198,14 +230,14 @@ <h2 class="h2">Multi-Post<br />Stories</h2>
     <section class="container-section2">
```

Comment:
- [ ] Please use the correct right margin for this section to match the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160193638-2559f76f-f759-47a4-9da7-57a0bd141a89.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160193609-df65d141-814c-4c30-af27-29222d27e1f1.png)